tests/test-integration/src/test/resources/RFC2616Base.xml

```diff
@@ -33,7 +33,7 @@
     <!-- Set handler Collection Structure                            -->
     <!-- =========================================================== -->
     <Set name="handler">
-      <New id="Handlers" class="org.eclipse.jetty.server.handler.HandlerCollection">
+      <New id="Handlers" class="org.eclipse.jetty.server.handler.HandlerList">
```

Comment:
@olamy the structure of this XML is a bit strange. Not sure why the two Contexts are not under the ContextHandlerCollection?

Reply:
No idea but I agree it's really strange and need to be fixed

Reply:
OK - will do in a separate PR